File: src/TIL_0905/kakao_04_fail.py
def solution(words, queries):
    answer = []

    for query in queries:
        
        i = 0

        left = True        
        l = ''
        r = ''
        idx = 0
        if query[0]=='?':
            while query[idx]=='?':
                l += query[idx]
                idx += 1
            r = query[idx:]
        else:
            left = False
            while query[idx]!='?':
                l += query[idx]
                idx += 1
            r = query[idx:]

        cnt = 0
        for word in words:
            if len(word)==len(query):
                if left:
                    if word[idx:]==r:
                        cnt += 1
                else:
                    if word[:idx]==l:
                        cnt += 1

        answer.append(cnt)

    return answer

File: src/TIL_0905/kakao_04_fail.java
package TIL_0905;
import java.util.*;
public class kakao_04_fail {
	public static void main(String[] args) {
		String[] words = { "a","b","frodo", "front", "frost", "frozen", "kakao", "frame" };
		String[] queries = { "?","fro??", "????o", "fr???", "fro???", "pro??", "??ont", "????o" };
		
		System.out.println(solution(words, queries));
	}
    
	public static int[] solution(String[] words, String[] queries) {
		int[] answer = new int[queries.length];
        return answer;
    }
}

class Trie {
	int count;
	Trie[] childList = new Trie[26];
	
	Trie() {
		childList = new Trie[26];
	}
}